backend/src/main/kotlin/app/ehrenamtskarte/backend/application/webservice/schema/create/Attachment.kt

```diff
@@ -0,0 +1,21 @@
+package app.ehrenamtskarte.backend.application.webservice.schema.create
+
+import app.ehrenamtskarte.backend.common.webservice.GraphQLContext
+import com.expediagroup.graphql.annotations.GraphQLIgnore
+import graphql.schema.DataFetchingEnvironment
+import java.lang.IllegalArgumentException
+import javax.servlet.http.Part
+
+data class UploadKey (val index: Int)
```

Comment:
I suspect that index allows referencing the received input stream globally right?

Reply:
yep